src/components/AddItem.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
I believe that we can remove this import and that it's not needed for these components.

Reply:
@rachael-t We can def get rid of the react import in our List and AddItem components- thanks for pointing that out! Do you know if we still need to import it in App.js or is it automatically imported there as well?

Reply:
I believe that it isn't needed with React 17 but I may be mistaken - we could have the mentors chime in on this as well?

Reply:
@rachael-t you're correct! This was a helper provided with [react 17](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html). 

It's worth mentioning that other projects/jobs you work on may not be using react 17, so always check what version of react you're using in `package.json`